event_routing_backends/processors/mixins/base_transformer.py

```diff
@@ -102,3 +102,16 @@ def transform(self):
                 )
 
         return self.transformed_event
+
+    def extract_username(self):
+        """
+        extract username edX event.
+
+        Returns:
+            username
```

Comment:
I think it should type of return value like `str`

Reply:
done